qi-doc/scribblings/qi.scrbl

```diff
@@ -9,6 +9,16 @@
 
 An embeddable, general-purpose language to allow convenient framing of programming logic in terms of functional @emph{flows}.
 
+Tired of writing long functional pipelines with nested syntax like this?
+@racketblock[(map _f (filter _g (vector->list _my-awesome-data)))]
+Then Qi is for you!
+@racketblock[(~>> (_my-awesome-data) vector->list (filter _g) (map _f))]
+But wait, there's more: Qi isn't just a turbo-charged threading language. It
+supports multiple values and a suite of other operators for describing
+computations:
+@racketblock[(define-flow average
+               (~> sep (-< + count) /))]
```

Comment:
The docs standardize on `△` rather than `sep` outside of the documentation of the form itself. Alternatively, we could assume the input is values here, making it:

```
(define-flow average
  (~> (-< + count) /))
```

... which might actually be better since it expresses the essential idea of average without involving lists.

Reply:
I'm ok with either; I had forgotten that the docs prefer the single-character versions. I'll probably go with the latter and a usage example.